lib/ledger_sync/adaptors/quickbooks_online/department/ledger_serializer.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module LedgerSync
+  module Adaptors
+    module QuickBooksOnline
+      module Department
+        class LedgerSerializer < QuickBooksOnline::LedgerSerializer
+          id
```

Comment:
I'm unfamiliar with the DSL. Out of curiosity, if we can do this with `id`, can we do it with attributes like `name` and `active` too?

Reply:
@mattgmarcus this is just a class function on the `QuickBooksOnline::LedgerSerializer` class given that most QBO serializers use IDs the same way.  We may change this given that we need to split up the serializers and deserializers anyway.